openssl-dynamic/src/main/c/ssl.c

```diff
@@ -678,21 +935,34 @@ TCN_IMPLEMENT_CALL(jint, SSL, getError)(TCN_STDARGS,
     return SSL_get_error(ssl_, ret);
 }
 
-// How much did SSL write into this BIO?
-TCN_IMPLEMENT_CALL(jint /* nbytes */, SSL, pendingWrittenBytesInBIO)(TCN_STDARGS,
-                                                                     jlong bio /* BIO * */) {
-    BIO *b = J2P(bio, BIO *);
+TCN_IMPLEMENT_CALL(jint /* nbytes */, SSL, bioCtrlPending)(TCN_STDARGS, jlong bio /* BIO* */) {
+    BIO* b = J2P(bio, BIO*);
 
     if (b == NULL) {
         tcn_ThrowException(e, "bio is null");
         return 0;
     }
 
-    UNREFERENCED_STDARGS;
-
     return BIO_ctrl_pending(b);
 }
 
+#ifdef OPENSSL_IS_BORINGSSL
```

Comment:
Personally, I think `BIO_ctrl` is insane, so not hooking into it seems pretty sound. But it's also possible they're appropriate here? (Didn't look at this PR.) I dunno. OpenSSL doesn't really use values, so it doesn't especially matter either way.

But my comment was just pointing out that `BIO_ctrl_wpending` and `BIO_wpending` are the exact same thing.